src/widget/wcoverartmenu.cpp

```diff
@@ -82,8 +86,38 @@ void WCoverArtMenu::slotChange() {
     coverInfo.source = CoverInfo::USER_SELECTED;
     coverInfo.coverLocation = selectedCoverPath;
     coverInfo.setImage(image);
-    qDebug() << "WCoverArtMenu::slotChange emit" << coverInfo;
-    emit coverInfoSelected(coverInfo);
+
+    QString coverArtCopyFilePath =
+            fileInfo.absoluteFilePath().left(
```

Comment:
```suggestion
            trackFileInfo.absoluteFilePath().left(
```
fileInfo can be null see above. Can this case happen? If yes we need to handle it. Otherwise assert that it not happens

Reply:
File info comes from the cover info's track location. As far as I know, Track location can not be empty because I was having assertion while I was working on the Track Suggestion Feature. I am 100% sure tho

Reply:
i don't think so. Just assert for it in the condition above and we will learn during testing if our assumption is correct.